tootanky/champion.py

```diff
@@ -339,3 +339,9 @@ def do_auto_attack(self, target, is_crit: bool = False):
 
     def reset_health(self):
         self.health = self.orig_base_stats.health + self.orig_bonus_stats.health
+
+    def has_item(self, item_name: str) -> bool:
+        return self.inventory.contains(item_name)
+
+    def get_item(self, item_name: str) -> bool:
+        return self.inventory.get_item(item_name)
```

Comment:
Not a big fan because it adds nothing.

Reply:
so  you are fine with 

```
champion.inventory.get_item()
champion.rune_page.get_rune()
```
instead of 
```
champion.get_item()
champion.get_rune()
```

Reply:
I think so, but that's not a big deal.